Docs/source/developers/amrex_basics.rst

```diff
@@ -0,0 +1,28 @@
+AMReX basics (excessively basic)
+================================
+
+WarpX is build on the Adaptive Mesh Refinement (AMR) library `AMReX <https://github.com/AMReX-Codes/amrex>`__. This section provides a very sporadic description of the main AMReX classes and concepts relevant for WarpX, that can serve as a reminder. Please read the AMReX basics `doc page <https://amrex-codes.github.io/amrex/docs_html/Basics.html>`__, of which this section is largely inspired.
+
+* ``amrex::Box``: Dimension-dependent lower and upper indices defining a rectangular volume in 3D (or surface in 2D) in the index space. ``Box`` is a lightweight meta-data class, with useful member functions.
+
+* ``amrex::BoxArray``: Collection of ``Box`` on a single AMR level. The information of which MPI rank owns which ``Box`` in a ``BoxArray`` is in ``DistributionMapping``.
+
+* ``amrex::FArrayBox``: Fortran-ordered array of floating-point ``amrex::Real`` elements defined on a ``Box``. A ``FArrayBox`` can represent scalar data or vector data, with ``ncomp`` components.
+
+* ``amrex::MultiFab``: Collection of `FAB` (= ``FArrayBox``) on a single AMR level. The concept of `ghost cells` is defined at the ``MultiFab`` level.
+
+* ``amrex::Particle``: Data for 1 particle. It contains default member variables stored as Array-Of-Struct (AoS) (often called `data`, they are the 3D position, the particle ID and the index of the CPU owning the particle) and extra variables (often called `attribs`, in WarpX they are the momentum, field on particle etc.), stored at Struct-Of-Arrays (SoA).
```

Comment:
the class called `amrex::Particle` only has the AoS data, so this is slightly misleading. I'd merge the above description of AoS versus SoA in at the end of the next section about `ParticleContainer` and skip documenting `amrex::Particle`.

Reply:
Thanks, that was fixed in the last commit